x-pack/plugins/cases/server/services/attachments/index.ts

```diff
@@ -42,15 +32,10 @@ import {
   injectAttachmentSOAttributesFromRefsForPatch,
 } from '../so_references';
 import type { SavedObjectFindOptionsKueryNode } from '../../common/types';
-import type { PersistableStateAttachmentTypeRegistry } from '../../attachment_framework/persistable_state_registry';
 import type { IndexRefresh } from '../types';
+import type { AttachedToCaseArgs, GetAttachmentArgs, ServiceContext } from './types';
+import { AttachmentGetter } from './operations/get';
 
-interface AttachedToCaseArgs extends ClientArgs {
```

Comment:
Moved a lot of types to the new `types.ts` file. Also removed a lot of `ClientArgs` references because the saved object client is no longer passed as a parameter to each operation but rather to the class constructor.

Hiding whitespace when reviewing these changes should make it a bit easier.